Tests/Crypto/BIP32/BIP32KeypairDerivationTests.swift

```diff
@@ -0,0 +1,96 @@
+import XCTest
+import FearlessUtils
+import IrohaCrypto
+
+class BIP32KeypairDeriviationTests: XCTestCase {
+
+    func testBIP32DerivationPath() throws {
+        try performTest(filename: "BIP32HDKD", keypairFactory: BIP32KeypairFactory(), useMiniSeed: false)
+    }
+
+    func testBIP32DerivationPathStandardData() throws {
```

Comment:
Consider to rename to `testBIP32DerivationPathForEtalonTestVectors`

Reply:
Good suggestion, I definitely struggled with naming
✅